man/systemd.network.xml

```diff
@@ -652,6 +652,14 @@
             <para>An address label.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>PreferredLifetime=</varname></term>
+          <listitem>
+            <para>A boolean. When IPv6 address's preferred lifetime expires, the address becomes
+            deprecated Which it holds during its preferred-lifetime. When true "forever"
+            else "zero". Defaults to true.</para>
```

Comment:
This description is hard to grok :(

I know that @teg requested this interface in #2166, but it seems risky. In particular `PreferredLifetime=1` is particularly misleading, since it means infinity. And *if* we want want to extend the setting in the future to allow setting the lifetime to a concrete value, it'll be hard to do. So what about the following instead:

> Allows the default "preferred lifetime" of the address to be overridden. Only two settings are accepted: &lt;literal>forever&lt;/literal>, which is the default and means that the address never expires, and &lt;literal>0&lt;/literal> which means that the address is considered immediately "expired" and will not be used, unless explicitly requested. A setting of <varname>PreferredLifetime=0</varname> is useful for addresses which are added to be used only by a specific application, which is then configured to use them explicitly.

And then of course the code should be updated to match this description...